.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # At least one of the code owners below will be required on each PR:
 
-* @markAtMicrosoft @smmatte @adclements @abmahdy @LukeSlev @iennae
+* @markAtMicrosoft @smmatte @adclements @abmahdy @LukeSlev @iennae @andreamah @xnkevinnguyen @vandyliu @nasadigital @isadorasophia @sagarmanchanda
```

Comment:
After some consultation, Stephane and I agree, at least for now, please remove interns from CODEOWNERS file.  Coaches should remain.  Also remove @LukeSlev.  Because this is a live code base, we'd like another set of eyes.  The rules are basically that 2 approves are required and one of them must be code owner

Reply:
Ok, we will update accordingly, Thanks

Reply:
Done!

Reply:
![byebye](https://user-images.githubusercontent.com/28886212/72852547-13877380-3c6c-11ea-8040-c83519ac5c60.gif)